Index: kernel-source/arch/arm/boot/dts/allwinner/sunxi-h3-h5.dtsi
===================================================================
--- kernel-source.orig/arch/arm/boot/dts/allwinner/sunxi-h3-h5.dtsi
+++ kernel-source/arch/arm/boot/dts/allwinner/sunxi-h3-h5.dtsi
@@ -599,7 +599,7 @@
 			pinctrl-names = "default";
 			pinctrl-0 = <&spi0_pins>;
 			resets = <&ccu RST_BUS_SPI0>;
-			status = "disabled";
+			status = "okay";
 			#address-cells = <1>;
 			#size-cells = <0>;
 		};
